molo/forms/tests/test_views.py

```diff
@@ -243,11 +244,9 @@ def test_show_results_option_ajax(self):
         self.assertContains(response, molo_form_page.title)
         self.assertContains(response, molo_form_page.introduction)
         self.assertContains(response, molo_form_field.label)
-
-        response = self.client.post(molo_form_page.url, {
+        response = self.client.post(molo_form_page.get_full_url(), {
             "ajax": 'True',
-            "article_page": self.article.pk,
```

Comment:
Why did you remove the article_page field here?

Reply:
it is not needed. Form is not attached to that article